src/main/java/nextstep/subway/domain/LineBridges.java

```diff
@@ -0,0 +1,56 @@
+package nextstep.subway.domain;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Embeddable;
+import javax.persistence.OneToMany;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+
+@Embeddable
+public class LineBridges {
+
+    @OneToMany(mappedBy = "line", cascade = CascadeType.ALL)
+    private List<LineBridge> lineBridges = new ArrayList<>();
+
+    public void add(LineBridge lineBridge) {
+        validateDuplicate(lineBridge);
+        validateNotingMatch(lineBridge);
+
+        repairSections(lineBridge);
+        lineBridges.add(lineBridge);
+    }
+
+    private void validateDuplicate(LineBridge lineBridge) {
+        Optional<LineBridge> duplicate = lineBridges.stream()
+                .filter(element -> element.isSame(lineBridge))
```

Comment:
```suggestion
                .filter(lineBridge::isSame)
```
메소드 참조를 활용해도 좋을것 같아요